### DISCUSSION TITLE
qsv sniff path value on Windows may show \\? format

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#2042] [Q&A]
Not sure if this is an issue, see the **Path** value. The [dunce](https://crates.io/crates/dunce) crate may be useful if this is something that should be handled.

![image](https://github.com/user-attachments/assets/a5c6863f-3a24-464d-8656-d01912f9a71b)

**Comment by rzmk** [comment]
Or maybe added as a separate path value for Windows.

**Reply by jqnatividad** [reply]
Thanks @rzmk ,
The `sniff` command is due for a major refactor (#1976).

Will be sure to add path testing to the CI tests.
